packages/components/src/components/FunctionChart.tsx

```diff
@@ -58,7 +152,15 @@ export const FunctionChart: React.FC<FunctionChartProps> = ({
     setMouseOverlay(NaN);
   }
   const signalListeners = { mousemove: handleHover, mouseout: handleOut };
-  let mouseItem = runForeign(fn, [mouseOverlay], environment);
+  let mouseItem: result<squiggleExpression, errorValue> = !!mouseOverlay
+    ? runForeign(fn, [mouseOverlay], {
+        sampleCount: 10000,
+        xyPointLength: 1000,
+      })
+    : {
+        tag: "Error",
+        value: { tag: "REExpectedType", value: "Expected float, got NaN" },
```

Comment:
This error message seems to be very confusing. If for some reason mouseOverlay is returning NaN, then I might just want to skip the render rather than return an error? When does this type of thing happen?

Reply:
> When does this type of thing happen?
It seems like a vega/react thing. It happens in two situations: one; when the cursor is outside the common range, and two, just randomly sometimes (from what I can tell)